### PR TITLE
Polygon_mesh_processing: Change should  to must/shall

### PR DESCRIPTION
**Pull request by afabri**
## Summary of Changes

The word _should_  is too weak for a requirement.

## Release Management

* Affected package(s): Polygon_mesh_processing documentation.

### CODE REVIEW

File: Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/distance.h [hunk 1]
```diff
@@ -611,8 +611,8 @@ double approximate_Hausdorff_distance(
  *
  * \cgalNamedParamsBegin
  *    \cgalParamBegin{vertex_point_map} the property map with the points associated to the vertices of `tm2`
- *      If this parameter is omitted, an internal property map for CGAL::vertex_point_t should be available in `TriangleMesh`
- *      and in all places where vertex_point_map is used.
+ *      If this parameter is omitted, an internal property map for `CGAL::vertex_point_t` `must be available in `TriangleMesh`
```
Comment (janetournois) [inline Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/distance.h]: there is an extra backqutote before "must"

File: Polygon_mesh_processing/doc/Polygon_mesh_processing/dependencies [hunk 2]
```diff
@@ -12,3 +12,4 @@ Surface_mesh_deformation
 AABB_tree
 Triangulation_2
 Spatial_sorting
+Random_numbers
```
Comment (lrineau) [inline Polygon_mesh_processing/doc/Polygon_mesh_processing/dependencies]: Doc error!
https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-4.13-Ic-30/logs_master/Polygon_mesh_processing.log
```
error: Tag file `/home/cgal-testsuite/cgal_doc_build/CGAL-4.13-Ic-30/doc/scripts/doc_master/doc_tags/Random_numbers.tag' does not exist or is not a file. Skipping it...
```

File: Polygon_mesh_processing/doc/Polygon_mesh_processing/dependencies [hunk 3]
```diff
@@ -12,3 +12,5 @@ Surface_mesh_deformation
 AABB_tree
 Triangulation_2
 Spatial_sorting
+Generator
```
Comment (sloriot) [inline Polygon_mesh_processing/doc/Polygon_mesh_processing/dependencies]: @afabri can you comment on why this is needed?

Comment (afabri) [reply Polygon_mesh_processing/doc/Polygon_mesh_processing/dependencies]: In order to get a link to ` Random`   for the named parameter [` random_seed`](https://doc.cgal.org/latest/Polygon_mesh_processing/group__pmp__namedparameters.html#PMP_random_seed)

**Comment by lrineau** [comment]
Successfully tested in [CGAL-4.13-Ic-35][*], and then merged.

  [*]: https://cgal.geometryfactory.com/CGAL/testsuite/results-4.13-Ic-35.shtml

**Pull request merged**